lib/util.js

```diff
@@ -1472,6 +1472,7 @@ module.exports = exports = {
   callbackify,
   debuglog,
   deprecate,
+  features: Symbol('features'),
```

Comment:
can the description be `node.util.features`

Reply:
I'm not sure what you mean